src/resource-scheduling/simulator.md

```diff
@@ -0,0 +1,87 @@
+# Simulator
+
+## Overview
+
+With its flexibility and significant benefits of reducing the time and the
+cost, the simulator plays an important role in studying and designing a
+computer architecture. It is often used to validate specific design schemes
+and evaluate the effectiveness of design schemes.
+
+## Workflow
+
+Since this book is mainly targeted towards the people who are interesting in
```

Comment:
This book? Are we talking about a book here?

Reply:
I guess so?

Reply:
How about: "Since our targeted audience are people who are interested in"